### ISSUE TITLE
[Bug] Installing k9s on Rocky 8 fails: "curl: option --fail-with-body: is unknown"

### ISSUE THREAD

**Issue by brnl** [webinstall/webi-installers#769]
## What were you trying to install (or what else went wrong)?
I was trying to install k9s on Rocky 8, which fails with the error `curl: option --fail-with-body: is unknown`.

The curl version on Rocky 8 is:
> curl 7.61.1 (x86_64-redhat-linux-gnu) libcurl/7.61.1 OpenSSL/1.1.1k zlib/1.2.11 brotli/1.0.6 libidn2/2.2.0 libpsl/0.20.2 (+libidn2/2.2.0) libssh/0.9.6/openssl/zlib nghttp2/1.33.0
> Release-Date: 2018-09-05
> Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtsp scp sftp smb smbs smtp smtps telnet tftp 
> Features: AsynchDNS IDN IPv6 Largefile GSS-API Kerberos SPNEGO NTLM NTLM_WB SSL libz brotli TLS-SRP HTTP2 UnixSockets HTTPS-proxy PSL 

The option `--fail-with-body` is only supported since curl version 7.76.0 (source: https://daniel.haxx.se/blog/2021/02/11/curl-fail-with-body/)

## What exactly did you do?
```sh
curl -sS https://webinstall.dev/k9s | bash
```

## What went wrong?

```
Bootstrapping Webi
    Downloading https://webinstall.dev/packages/webi/webi.sh
        to ~/.local/bin/webi
curl: option --fail-with-body: is unknown
curl: try 'curl --help' or 'curl --manual' for more information
    failed to download (curl) 'https://webinstall.dev/packages/webi/webi.sh'
    curl --fail-with-body -sSL -# -A 'webi/curl x86_64/unknown Linux/4.18.0-477.27.1.el8_8.x86_64 gnu' 'https://webinstall.dev/packages/webi/webi.sh' -o '/home/ce-beheer/.local/bin/webi.part'
    curl 7.61.1 (x86_64-redhat-linux-gnu) libcurl/7.61.1 OpenSSL/1.1.1k zlib/1.2.11 brotli/1.0.6 libidn2/2.2.0 libpsl/0.20.2 (+libidn2/2.2.0) libssh/0.9.6/openssl/zlib nghttp2/1.33.0
Release-Date: 2018-09-05
Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtsp scp sftp smb smbs smtp smtps telnet tftp 
Features: AsynchDNS IDN IPv6 Largefile GSS-API Kerberos SPNEGO NTLM NTLM_WB SSL libz brotli TLS-SRP HTTP2 UnixSockets HTTPS-proxy PSL 
```
## Which OS did you try on?

- [ ] Mac
- [ ] Windows
- [x] Linux

## What type of computer (i.e. laptop, desktop, Raspberry Pi)?

- [ ] Laptop or Desktop (amd64 / x86-64)
- [ ] Raspberry Pi (ARM-64)
- [x] Other (please specify, if you know): Rocky Linux 8 VM

**Comment by coolaj86** [comment]
The improved error handling causes errors... 🤦‍♂️

If you care to PR the 3 affected files are in `_webi`.

Otherwise I'll fix this within the hour.

Thanks for the report!

**Comment by coolaj86** [comment]
Fixed via #770 which is already deployed live via the `hotfix` branch.
